Gems/WhiteBox/Code/Source/EditorWhiteBoxComponentMode.h

```diff
@@ -95,13 +100,24 @@ namespace WhiteBox
         bool m_restoreModifierHeld = false;
 
         AzToolsFramework::ViewportUi::ClusterId
-            m_modeSelectionClusterId; //!< Viewport UI cluster for changing sub mode.
+            m_transformClusterId; 
+        AzToolsFramework::ViewportUi::ButtonId
+            m_transformTranslateButtonId; 
+        AzToolsFramework::ViewportUi::ButtonId
+            m_transformRotateButtonId; 
+        AzToolsFramework::ViewportUi::ButtonId
+            m_transformScaleButtonId; 
+
+        AzToolsFramework::ViewportUi::ClusterId
+            m_modeSelectionClusterId;  //! Viewport UI cluster for changing sub mode.
+        AzToolsFramework::ViewportUi::ButtonId
+            m_defaultModeButtonId;     //! Id of the Viewport UI button for default mode.
         AzToolsFramework::ViewportUi::ButtonId
-            m_defaultModeButtonId; //!< Id of the Viewport UI button for default mode.
+            m_edgeRestoreModeButtonId; //! Id of the Viewport UI button for edge restore mode.
         AzToolsFramework::ViewportUi::ButtonId
-            m_edgeRestoreModeButtonId; //!< Id of the Viewport UI button for edge restore mode.
+            m_transformModeButtonId;   //! Id of the Viewport UI button for transform mode.
         AZ::Event<AzToolsFramework::ViewportUi::ButtonId>::Handler
-            m_modeSelectionHandler; //!< Event handler for sub mode changes.
+            m_modeSelectionHandler;    //! Event handler for sub mode changes.
```

Comment:
update: If you're removing the `<`, please move the comment above the variable declaration

e.g.

```
//! Event handler for sub mode changes.
AZ::Event<AzToolsFramework::ViewportUi::ButtonId>::Handler m_modeSelectionHandler;
```

If you do this can you move everything back on to its own line as above, thanks!